### PR TITLE
test/ffmpeg/gstreamer: add vpp/sharpen r2r test

### PR DESCRIPTION
**Pull request by Hangjie22Coder**

### CODE REVIEW

File: test/ffmpeg-qsv/vpp/sharpen.py [hunk 1]
```diff
@@ -31,3 +32,10 @@ def test(self, case, level):
         return
 
     self.vpp()
+
+  @slash.parametrize(*gen_vpp_sharpen_parameters(spec_r2r))
+  def test_r2r(self, case, level):
+    vars(self).update(spec[case].copy())
```
Comment (uartie) [inline test/ffmpeg-qsv/vpp/sharpen.py:38]: Should be `spec_r2r`

File: test/ffmpeg-vaapi/vpp/sharpen.py [hunk 2]
```diff
@@ -32,3 +33,10 @@ def test(self, case, level):
         return
 
     self.vpp()
+
+  @slash.parametrize(*gen_vpp_sharpen_parameters(spec_r2r))
+  def test_r2r(self, case, level):
+    vars(self).update(spec[case].copy())
```
Comment (uartie) [inline test/ffmpeg-vaapi/vpp/sharpen.py:39]: Should be `spec_r2r`

File: test/gst-msdk/vpp/sharpen.py [hunk 3]
```diff
@@ -31,3 +32,10 @@ def test(self, case, level):
 
     vars(self).update(case = case, level = level)
     self.vpp()
+
+  @slash.parametrize(*gen_vpp_sharpen_parameters(spec_r2r))
+  def test_r2r(self, case, level):
+    vars(self).update(spec[case].copy())
```
Comment (uartie) [inline test/gst-msdk/vpp/sharpen.py:38]: Should be `spec_r2r`

File: test/gst-va/vpp/sharpen.py [hunk 4]
```diff
@@ -34,3 +35,10 @@ def test(self, case, level):
         return
 
     self.vpp()
+
+  @slash.parametrize(*gen_vpp_sharpen_parameters(spec_r2r))
+  def test_r2r(self, case, level):
+    vars(self).update(spec[case].copy())
```
Comment (uartie) [inline test/gst-va/vpp/sharpen.py:41]: Should be `spec_r2r`

File: test/gst-vaapi/vpp/sharpen.py [hunk 5]
```diff
@@ -34,3 +35,10 @@ def test(self, case, level):
         return
 
     self.vpp()
+
+  @slash.parametrize(*gen_vpp_sharpen_parameters(spec_r2r))
+  def test_r2r(self, case, level):
+    vars(self).update(spec[case].copy())
```
Comment (uartie) [inline test/gst-vaapi/vpp/sharpen.py:41]: Should be `spec_r2r`

**Review by uartie — CHANGES_REQUESTED** [review]

File: test/gst-vaapi/vpp/sharpen.py (hunk 5 above)
Comment (Hangjie22Coder) [reply test/gst-vaapi/vpp/sharpen.py]: Thanks for the review, updated.

**Review by uartie — APPROVED** [review]

**Pull request merged**